release-process.md

```diff
@@ -319,4 +319,17 @@ Once everything is working (website docs, website changes) create an announcemen
 and then send an e-mail to the mailing list. To create an announcement, create a post under
 `news/_posts` and then run `jekyll build`.
 
+<h4>Update a test suite depending on old releases</h4>
```

Comment:
I think we can say "HiveExternalCatalogVersionsSuite" here instead of "a test suite" to be more direct. If it's many test suites later, we can change this.

Reply:
Yep.